src/components/ShowBooks.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Book from './Home';
+import Form from './Form';
+
+const books = [
+  { id: 1, title: 'The Hunger Games', author: 'Suzanne Collins' },
+  { id: 2, title: 'Dune', author: 'Frank Herbert' },
+  { id: 2, title: 'Capital in the Twenty-First Century', author: 'Suzanne Collins' },
+];
+
+const ShowBooks = () => (
+  <div>
+    {books.map((book) => (
+      <Book key={book.id} title={book.title} author={book.author} />
+    ))}
+    <Form />
+  </div>
+);
+
+export default ShowBooks;
```

Comment:
- [ ] Good job creating the `Form` and `Book` components as required. However, if you read the requirements carefully you will see that we need to create a third component which will contain the entire list of the books which are added to the app. It is the second requirement in the screenshot below. Kindly check the requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md)

![image](https://github.com/rbd3/bookstore/assets/81932478/2bdd3509-0120-4b98-b81c-74e6613d58f5)

So instead of displaying the books using the `ShowBooks` component, I would advise that you create a component to hold the entire book list e.g you can call it `BookList`  and import it into the `ShowBooks` component and use it to display the list of books as per requirement. 
This means that tje `ShowBooks` component will import three components instead of two.